webapp/tests/karma/js/enketo/widgets/phone-widget.spec.ts

```diff
@@ -101,13 +101,4 @@ describe('Enketo: Phone Widget', () => {
 
     expect(input.val()).to.equal(valid);
   });
-
-  it('should not modify non-phone fields', () => {
```

Comment:
This test started failing when I updated the `phone-widget` to extend the Enketo `Widget` class because the super constructor in the `Widget` class tries to reference the element parameter that is passed in and in the case of this test, `$(phoneWidget.selector)[0]` will be `undefined` since there is nothing that matches the selector in the HTML file created by `buildHtml('other')`.

Ultimately, I concluded that this seemed to be an odd test to have anyway since it was basically asserting that, if nothing is passed in, then the original html doc is not affected....  Maybe I am missing something here though?

Reply:
Yeah, maybe.  Seems like it would have been more clear to just check:
```.js
expect($(phoneWidget.selector)).isEmpty();
```

Reply:
🤷🏻‍♀️ I think having a simple selector test makes sense, if I'd be creating a new widget I'd consider that necessary.